README.md

```diff
@@ -8,28 +8,40 @@ btcwallet is a daemon handling bitcoin wallet functionality for a
 single user.  It acts as both an RPC client to btcd and an RPC server
 for wallet clients and legacy RPC applications.
 
-The wallet file format is based on
-[Armory](https://github.com/etotheipi/BitcoinArmory) and provides a
-deterministic wallet where all future generated private keys can be
-recovered from a previous wallet backup.  Unencrypted wallets are
-unsupported and are never written to disk.  This design decision has
-the consequence of generating new wallets on the fly impossible: a
-client is required to provide a wallet encryption passphrase.
+Public and private keys are derived using the heirarchical
+deterministic format described by
+[BIP0032](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki).
+Unencrypted private keys are not supported and are never written to
+disk.  btcwallet uses the
+`m/44'/<coin type>'/<account>'/<branch>/<address index>`
+HD path for all derived addresses, as described by
+[BIP0044](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki).
+
+Due to the sensitive nature of public data in a BIP0032 wallet,
+btcwallet provides the option of encrypting not just private keys, but
+public data as well.  This is intended to thwart privacy risks where a
+wallet file is compromised without exposing all current and future
```

Comment:
Perhaps something like:

This is intended to thwart privacy risks where a wallet file is compromised without exposing all current and future addresses *(public keys)* managed by the wallet.  While access to this information would not allow an attacker to spend or steal coins, it does mean they could track all transactions involving your addresses and therefore know your exact balance.